lib/trader.js

```diff
@@ -62,6 +62,11 @@ Trader.prototype.syncLogs = function syncLogs () {
   // Get last seen timestamp from server
   this.request({ path: '/logs', method: 'get' })
     .then(data => data.body)
+  // Delete log files that are two or more days old
+    .then(() => {
+      let today = new Date()
+      return logs.removeLogFiles(today.setDate(today.getDate() - 2))
```

Comment:
I don't think this will work. ``getDate()`` gets the day of the month, not a date object. It doesn't seem like this code was tested.

Reply:
Hmm, have you tried it in your computer? It's true that `getDate()` returns the day of the month which is fed to `setDate()` (effectively setting the date to two days ago) which in turn returns a Unix timestamp.

<img width="355" alt="nodejs-date" src="https://user-images.githubusercontent.com/41801303/55383945-4ca9f800-5521-11e9-9ed1-9baab30c1c40.png">